test.c

```diff
@@ -82,6 +109,11 @@ int main(int argc, char *argv[]) {
 	dropout = atof(getenv("DROPOUT") ?: "0");
 	dropfrom = atoi(getenv("DROPFROM") ?: "0");
 	dropto = atoi(getenv("DROPTO") ?: "0");
+	reorder = atof(getenv("REORDER") ?: "0");
+	reorder_dist = atoi(getenv("REORDER_DIST") ?: "10");
```

Comment:
Omitting the ternary operator's first parameter is a GNU extension and should be avoided for portable code. No clue how MSVC would handles this. This probably only works because right now we still compile meshlink with mingw, which we'll try to change to MSVC in the future.